src/app/Plugins/Weather/Components/weather/weather.component.html

```diff
@@ -1,5 +1,5 @@
 <div id="completeLayout" fxLayout="column" fxLayoutAlign="space-around center" class="mat-elevation-z2">
-
+  <mat-progress-bar *ngIf="!locationSelected" mode="indeterminate"></mat-progress-bar>
```

Comment:
add "&& !noLocationFound" because the progress bar should not be there if no location found